vocabularies/report-detail-type.ttl

```diff
@@ -10,26 +10,71 @@
         skos:ConceptScheme ;
     dcterms:created "2020-04-29"^^xsd:date ;
     dcterms:creator <http://linked.data.gov.au/org/gsq> ;
-    dcterms:modified "2020-04-29"^^xsd:date ;
+    dcterms:modified "2020-07-03"^^xsd:date ;
     dcterms:publisher <http://linked.data.gov.au/org/gsq> ;
     dcterms:source "Compiled by the Geological Survey of Queensland" ;
     skos:definition "Miscellaneous details relating to reports and documents."@en ;
     skos:prefLabel "Report Detail Type"@en ;
-    skos:hasTopConcept rptdt:project-name,
-        rptdt:placeholder .
+    skos:hasTopConcept rptdt:address,
+            rptdt:comments,
+            rptdt:contact,
+            rptdt:contracting-company,
+            rptdt:email,
+            rptdt:operating-company,
+            rptdt:phone,
+            rptdt:project-name .

```

Comment:
Please add "Activities undertaken" as features in PGGD-04
https://rawcdn.githack.com/geological-survey-of-queensland/gsq-lodgement-portal/7137f8bd7e0720a27892fc637d24101f7147f830/pggd04.html

This could also cover "Activities to be undertaken" in PGGD-03?
https://rawcdn.githack.com/geological-survey-of-queensland/gsq-lodgement-portal/7137f8bd7e0720a27892fc637d24101f7147f830/pggd03.html

Please also add "Did any incidents occur?" (choose a better name) from PGGD-04

Reply:
@DavidCrosswellGSQ 

Added concept _reportable-incident_ for  "Did any incidents occur?" flag.
Added altLabels to _comments_ concept to include "Activities undertaken" and "Activities to be undertaken", as reflected in the definition and generalised intent of this concept to capture planned activities and activity summaries.

Reply:
I really like the reportable-incident concept.
_Comments_, for me, is a different concept to '_Activities undertaken/to be undertaken_".
Comments would cover remarks and other unstructured data.
I think that the business will want to write a query such as "Return all activities undertaken when report type = PGGD-04".

Reply:
Good Point. I've changed the definition for comments to be more general commentary.
Added a concept simply Labelled "Activities" so it is independent of planned or actual. so you can query
- Return all activities when report type = PGGD-04 (find actual activities), or
- Return all activities when report type = PGGD-03 or PGGD-04 (compare planned vs actual)